### PR TITLE
chore: refreshed rfc-0190/mempool

### PR DESCRIPTION
**Pull request by agubarev**
Description
---
Added table of contents, rephrased some sentences to reduce tautology and make the read a bit more concise.

Motivation and Context
---
To review for possibly outdated material.

How Has This Been Tested?
---

### CODE REVIEW

File: src/RFC-0190_Mempool.md [hunk 1]
```diff
@@ -63,119 +80,118 @@ The Mempool is used for storing and managing unconfirmed [transaction]s.
...
 ### Prioritizing Unconfirmed Transactions
 
-The maximum storage capacity used for storing unconfirmed transactions by the Mempool and each of its component pools 
-can be configured. If a new transaction is received and the storage capacity limits have been reached, then 
-transactions are prioritized according to the transaction priority metric. The transaction priority metric consist 
-of the fee per gram of the UTXOs being spent by the transaction and should be applied to determine the priority 
-of each transaction in the Mempool. As the allocated storage space of the Mempool becomes limited, the transactions 
-with the highest priority are kept in the pool. The lowest priority transactions are discarded to make room for 
-higher priority incoming transactions.
+The maximum storage capacity used for storing unconfirmed transactions by the Mempool and each of its component pools
```
Comment (sdbondi) [inline src/RFC-0190_Mempool.md:102]: I think we should first describe the tx priority algo in this section and then mention what happens if the storage capacity is exceeded.

https://github.com/tari-project/tari/blob/development/base_layer/core/src/mempool/priority/prioritized_transaction.rs#L41

File: src/theme/book.js [hunk 2]
```diff
@@ -340,6 +340,23 @@ function playpen_text(playpen) {
         }
     });
 
+    // Toggle sidebar
```
Comment (stringhandler) [inline src/theme/book.js:343]: Need to move the non-RFC changes to a different PR

**Review by stringhandler — COMMENTED** [review]
NACK

ACK on the RFC, but CSS changes must be removed

File: src/theme/book.js (hunk 2 above)
Comment (agubarev) [reply src/theme/book.js:343]: Ah, my bad, I've branched it off the restyled branch.

File: src/RFC-0190_Mempool.md (hunk 1 above)
Comment (sdbondi) [reply src/RFC-0190_Mempool.md:102]: Actually, that might be overkill on the detail :smile: 

I think start with something like this
>transactions are prioritized by firstly the sum of fees of all UTXOs contained, followed by the length of time they have been in the mempool

and then mention what happens when the max storage cap is reached

File: src/RFC-0190_Mempool.md [hunk 3]
```diff
@@ -1,10 +1,27 @@
-# Mempool
+# RFC-0190/Mempool
+![status: updated](theme/images/status-stable.svg)
```
Comment (CjS77) [inline src/RFC-0190_Mempool.md:2]:
```suggestion
![status: stable](theme/images/status-stable.svg)
```

File: src/RFC-0190_Mempool.md [hunk 4]
```diff
@@ -48,8 +65,8 @@ technological merits of the potential system outlined herein.
 
 ## Goals
 
-This document will introduce the Tari [base layer] [Mempool] that consists of an [Unconfirmed Pool], and [Reorg Pool].
-The Mempool is used for storing and managing unconfirmed [transaction]s.
+This document will introduce the Tari [base layer] [Mempool] that consists of an [Unconfirmed Pool] and [Reorg Pool].
+The Mempool is used for storing and managing unconfirmed [transactions][transaction].
```
Comment (CjS77) [inline src/RFC-0190_Mempool.md:69]: nice. Been looking for the right syntax for this for ages

**Review by CjS77 — COMMENTED** [review]
**Bold**  _and_ UPPERCASE is overkill imho, but going to leave this as a nit

**Comment by CjS77** [comment]
ACK

**Review by CjS77 — APPROVED** [review]

**Pull request merged**